bluepyopt/electrical/celltemplate.py

```diff
@@ -194,17 +193,23 @@ def run_protocols(self, protocols, param_values=None):
 
     def __str__(self):
         """Return string representation"""
+        INDENT = ' ' * 2
+        lines = []
 
-        content = '%s:\n' % self.name
+        def out(indent_count, string):
```

Comment:
We should be very careful to define functions this way. These are not pickable (without an apparent workaround).